### ISSUE TITLE
Add some APIs to list the content of the zip file

### ISSUE THREAD

**Issue by jixingcn** [jixingcn/ZipPlatformFile#2]
more APIs:

```cpp
static IZipPlatformFile* GetZipPlatformFile(UObject* WorldContextObject);
static void FindDirectories(UObject* WorldContextObject, TArray<FString>& FoundDirectories, FString Directory, FString FileExtension, bool bRecursive = true)
static void FindFiles(UObject* WorldContextObject, TArray<FString>& FoundFiles, FString Directory, FString FileExtension, bool bRecursive = true)
```

**Comment by eldany1972** [comment]
You are the best!

**Comment by jixingcn** [comment]
Fixed in 9266b107c746751135b28e68f73b91f6a903469e

```cpp
static void FindFiles(UObject* WorldContextObject, TArray<FString>& FoundFiles, FString Directory, FString FileExtension, bool bRecursive = true);
static void FindDirectories(UObject* WorldContextObject, TArray<FString>& FoundDirectories, FString Directory);
```
